.eslintrc

```diff
@@ -0,0 +1,80 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jasmine": true,
+    "jquery": true,
+    "mocha": true,
+    "node": true
+  },
+  "extends": "eslint:recommended",
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "sourceType": "module"
+  },
+  "rules": {
+    "curly": "error",
+    "eqeqeq": "error",
+    "guard-for-in": "error",
+    "no-bitwise": "error",
+    "no-buffer-constructor": "error",
+    "no-caller": "error",
+    "no-console": "off",
+    "no-extra-boolean-cast": "off",
+    "no-undef": "error",
+    "no-unused-vars": [
+      "error",
+      {
+        "args": "none"
+      }
+    ],
+    "no-useless-escape": "off",
+    "semi": [
+      "error",
+      "always"
+    ],
+    "quotes": [
+      "error",
+      "single",
+      {
+        "allowTemplateLiterals": true
+      }
+    ]
+  },
+  "overrides": [
+    {
+      "files": [
+        "**/tests/**",
+        "**/test/**"
+      ],
+      "globals": {
+        "_": true,
+        "after": true,
+        "afterAll": true,
```

Comment:
Again, should these testing globals be in individual testing directories instead of here? Or do you think it's better to smush them all together at the top level like this? Does smushing them together require the `env` settings above, or can we put those in the right places and leave these here if you prefer them here?

Reply:
I prefer them here smushed together and the fact that I have specified them to only apply to `**/tests/**` folders makes them not to be applicable everywhere